src/core/src/database/lmdb.rs

```diff
@@ -0,0 +1,399 @@
+/*
+ * Copyright 2018 Intel Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ------------------------------------------------------------------------------
+ */
+
+use std::collections::HashMap;
+use std::path::Path;
+
+extern crate lmdb_zero as lmdb;
+
+use database::DatabaseError;
+
+const DEFAULT_SIZE: usize = 1 << 40; // 1024 ** 4
```

Comment:
** denotes _to the power_ here? Why not use a caret(^)?

Reply:
Both ** and ^ mean exponent